### PR TITLE
Handle acdd attribute project in format "Some Project Name (SPN)"

### PR DESCRIPTION
**Pull request by TAlonglong**
**Summary**:

**Related issue**: 
closses #247 

**Suggested reviewer(s)**:

**Reviewer checklist**:

- [x] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
- [x] 100% test coverage of new code - meaning:
    - [x] The overall test coverage increased or remained the same as before
    - [x] Every function is accompanied with a test suite
    - [x] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
    - [x] Functions with optional arguments have separate tests for all options
- [x] Examples are supported by doctests
- [x] All tests are passing
- [x] All names (e.g., files, classes, functions, variables) are explicit
- [x] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

**Comment by TAlonglong** [comment]
The tests are failing due to wrong format of the project attribute according to the new setup. Will fix this when we have agreed on the details.

**Comment by TAlonglong** [comment]
These tests are also failing. Blocked by https://github.com/metno/mmd/pull/217

**Review by johtoblan — APPROVED** [review]
LGTM

**Comment by mortenwh** [comment]
For the documentation - maybe the update could be done here. The project field in `mmd_elements.yaml` (https://github.com/metno/py-mmd-tools/blob/75c1d2893f2cd36ab50ca4043929709dcb730766/py_mmd_tools/mmd_elements.yaml#L447) needs to be updated, and then the DMH should be checked. Could be done as a separate issue as well but it is kind of urgent if the solution is not backward compatible.

**Comment by TAlonglong** [comment]
OK, so the short_name section should be remove. And update the long_name description.

**Comment by TAlonglong** [comment]
OK, I think I have it OK. But please have a look to ensure the implementation is as expected.

File: py_mmd_tools/mmd_elements.yaml [hunk 1]
```diff
@@ -448,17 +448,14 @@ project:
   maxOccurs: unbounded
   minOccurs: '0'
   short_name:
-    acdd_ext:
-      project_short_name:
-        separator: ';'
-        comment: Semicolon separated list.
-        description: Short version of the project name.
+    separator: ','
+    comment: Comma separated list.
```
Comment (mortenwh) [inline py_mmd_tools/mmd_elements.yaml:452]:
```suggestion
```

File: py_mmd_tools/mmd_elements.yaml [hunk 2]
```diff
@@ -448,17 +448,14 @@ project:
...
     acdd:
       project:
-        separator: ';'
-        comment: Semicolon separated list.
-        description: The name of the project(s) principally responsible for originating this data. Multiple projects can be separated by commas, as described under Attribute Content Guidelines. Examples; 'PATMOS-X', 'Extended Continental Shelf Project'. # If each substring includes a keyword in parantheses, the content within the paranthesis is interpreted as the short name for the project while the rest is the long name, e.g., 'Nansen Legacy (NLEG)'. --- this requires a modification of nc_to_mmd.py...
+        separator: ','
+        comment: Comma separated list.
+        description: The name of the project(s) principally responsible for originating this data in the format <long projectname> (<short project name>). Multiple projects can be separated by commas, as described under Attribute Content Guidelines. Examples; 'PATMOS-X', 'Extended Continental Shelf Project', 'LONG NAME (SN)'. # If each substring includes a keyword in parantheses, the content within the paranthesis is interpreted as the short name for the project while the rest is the long name, e.g., 'Nansen Legacy (NLEG)'.
```
Comment (mortenwh) [inline py_mmd_tools/mmd_elements.yaml:458]:
```suggestion
```

File: py_mmd_tools/mmd_elements.yaml [hunk 3]
```diff
@@ -448,17 +448,14 @@ project:
...
-    acdd_ext:
-      project_short_name:
-        separator: ';'
-        comment: Semicolon separated list.
-        description: Short version of the project name.
+    separator: ','
+    comment: Comma separated list.
   long_name:
     acdd:
       project:
```
Comment (mortenwh) [inline py_mmd_tools/mmd_elements.yaml:455]:
```suggestion
```

File: py_mmd_tools/mmd_elements.yaml [hunk 4]
```diff
@@ -448,17 +448,14 @@ project:
   maxOccurs: unbounded
   minOccurs: '0'
```
Comment (mortenwh) [inline py_mmd_tools/mmd_elements.yaml:449]:
```suggestion
  minOccurs: '0'
  acdd: 
      project:
          separator: ','
          comment: Comma separated list.
          description: The name of the project(s) principally responsible for originating this data in the format <long projectname> (<short project name>). Multiple projects can be separated by commas, as described under Attribute Content Guidelines. Examples; 'PATMOS-X', 'Extended Continental Shelf Project' becomes 'Extended Continental Shelf Project (PATMOS-X)'. I.e., if each substring includes a keyword in parantheses, the content within the paranthesis is interpreted as the short name for the project while the rest is the long name, e.g., 'Nansen Legacy (NLEG)'.
```

**Review by mortenwh — CHANGES_REQUESTED** [review]

**Comment by TAlonglong** [comment]
var en liten detalj i get_project som manglet. Fikset det og.

**Comment by mortenwh** [comment]
`tests/test_nc_to_mmd.py::test_create_mmd_1 - assert False is True`

I don't understand why the online tests are passing but I'll try to figure out of it...

**Review by mortenwh — APPROVED** [review]

**Pull request merged**